eng/Versions.props

```diff
@@ -144,6 +144,7 @@
   </PropertyGroup>
   <PropertyGroup>
     <!-- Dependencies from https://github.com/aspnet/AspNetCore -->
+    <MicrosoftAspNetCoreAppRefPackageVersion>8.0.0-preview.5.23258.6</MicrosoftAspNetCoreAppRefPackageVersion>
```

Comment:
This will need to get added to `Version.Details.xml` as well or it won't get auto-updated

Reply:
I thought it's already there.

```xml
<Dependency Name="Microsoft.AspNetCore.App.Ref" Version="8.0.0-preview.5.23258.6">
  <Uri>https://github.com/dotnet/aspnetcore</Uri>
  <Sha>80fea08dfb3edd85b114b1f3256f37d0a3bec678</Sha>
</Dependency>
```

Reply:
You mean the one that's already there?
https://github.com/dotnet/sdk/blob/f2f1006505b1397bb2042d5b720bca6fdb2293b4/eng/Version.Details.xml#L163

Reply:
Oh, weird that it wasn't already in here then. Never mind